content/news/2023-07-07-bevy-0.11/index.md

```diff
@@ -17,11 +17,146 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
+    // Has an implied CoreSchedule::Startup schedule
+    .add_startup_systems((a, b))
+    // Startup system variant 2.
+    // Has an implied default StartupSet::Startup base set
+    // Has an implied CoreSchedule::Startup schedule
+    .add_systems((a, b).on_startup())
+    // Startup system variant 3.
+    // Has an implied default StartupSet::Startup base set
+    .add_systems((a, b).in_schedule(CoreSchedule::Startup))
+    // Update system variant 1.
+    // `CoreSet::Update` base set and `CoreSchedule::Main` are implied
+    .add_system(c)
+    // Update system variant 2 (note the add_system vs add_systems difference)
+    // `CoreSet::Update` base set and `CoreSchedule::Main` are implied
+    .add_systems((d, e))
+    // No implied default base set because CoreSchedule::FixedUpdate doesn't have one
+    .add_systems((f, g).in_schedule(CoreSchedule::FixedUpdate))
+    // `CoreSchedule::Main` is implied, in_base_set overrides the default CoreSet::Update set
+    .add_system(h.in_base_set(CoreSet::PostUpdate))
+    // This has no implied default base set
+    .add_systems(enter_menu.in_schedule(OnEnter(AppState::Menu)))
+    // This has no implied default base set
+    .add_systems(exit_menu.in_schedule(OnExit(AppState::Menu)))
+```
+
+</details>
+
+Note that normal "system sets" still exist! You can still use sets to organize and order your systems:
+
+```rust
+app.add_systems(Update, (
+    (walk, jump).in_set(Movement),
+    collide.after(Movement),
+))
+```
+
+The `configure_set` API has also been adjusted for parity:
+
+```rust
+// before
+app.configure_set(Foo.after(Bar).in_schedule(PostUpdate))
+// after
+app.configure_set(PostUpdate, Foo.after(Bar))
+```
+
+## Nested System Tuples and Chaining
+
+<div class="release-feature-authors">authors: @cart</div>
+
+It is now possible to infinitely nest tuples of systems in a `.add_systems` call!
+
+```rust
+app.add_systems(Update, (
+    (a, (b, c, d, e), f),
+    (g, h),
+    i
+))
+```
+
+At first glance, this might not seem very useful. But in combination with per-tuple configuration, it allows you to easily and cleanly express schedules:
+
+```rust
+app.add_systems(Update, (
+    (attack, defend).in_set(Combat).before(check_health)
+    check_health,
+    (handle_death, respawn).after(check_health)
+))
+```
+
+`.chain()` has also been adapted to support arbitrary nesting! The ordering in the example above could be rephrased like this:
+
+```rust
+app.add_systems(Update,
+    (
+        (attack, defend).in_set(Combat)
+        check_health,
+        (handle_death, respawn)
+    ).chain()
+)
+```
+
+This will run `attack` and `defend` first (in parallel), then `check_health`, then `handle_death` and `respawn` (in parallel).
+
+This allows for powerful and expressive "graph-like" ordering expressions:
+
+```rust
+app.add_systems(Update,
+    (
+        (a, b, c).chain(),
+        (d, e),
+    ).chain()
+)
+```
+
+This will run a, then b, then c, then run d _and_ e in parallel.
```

Comment:
I don't see the use of this example. Isn't the ordering the same without the first `chain`?

Reply:
Hmmm yeah the intent was to illustrate the value of nested chaining but I guess this isn't the best illustrator of that.

Reply:
I've made the nested chaining example a bit more interesting / something that can't be trivially flattened out into a single chain.